client/disputes/new-evidence/shipping-details.tsx

```diff
@@ -12,19 +12,33 @@ import { TextControl } from 'wcpay/components/wp-components-wrapped';
 interface ShippingDetailsProps {
 	dispute: any;
 	readOnly?: boolean;
+	onShippingDetailsChange?: ( evidence: any ) => void;
 }
 
 const ShippingDetails: React.FC< ShippingDetailsProps > = ( {
 	dispute,
 	readOnly = false,
+	onShippingDetailsChange,
 } ) => {
+	const [ localEvidence, setLocalEvidence ] = React.useState(
+		dispute.evidence || {}
+	);
+
+	React.useEffect( () => {
+		setLocalEvidence( dispute.evidence || {} );
+	}, [ dispute ] );
```

Comment:
Why is it necessary to duplicate the state from the parent component into this child component? Could this component just provide the updated values to its parent via `onShippingDetailsChange`, without having to duplicate the `dispute.evidence` values?
(a similar question also comes up for `client/disputes/new-evidence/product-details.tsx`)